src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_LENGTH = 10;
+    private K[] keys;
+    private V[] values;
+    private int size = 0;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[MAX_LENGTH];
+        values = (V[]) new Object[MAX_LENGTH];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < MAX_LENGTH; i++) {
+            if (keys[i] != null && keys[i].equals(key) || keys[i] == key && values[i] != null) {
```

Comment:
Here we may use same structure as in get method:
```suggestion
            if (keys[i] != null && keys[i].equals(key) || keys[i] == key) {
```

Reply:
If I remove it, two tests will fail.
![image](https://github.com/mate-academy/jv-storage/assets/80032382/92ec1cfa-69a5-4155-9a40-9c3871cca6d4)